eng/pipelines/templates/jobs/smoke.tests.yml

```diff
@@ -159,15 +159,15 @@ jobs:
           timeoutInMinutes: 5
 
         - pwsh: |
```

Comment:
You will want to do something like https://github.com/Azure/azure-sdk-for-js/blob/main/eng/pipelines/templates/steps/build.yml#L81 to get the list of artifacts we want to shop it to then only update those packages.

Reply:
@ckairen this still isn't don't exactly what I expect it should `parameters.Artifact.name` assumes that there is ever only one Artifact but it a lot of contexts that is a list so that might fail in any such cases, consider the nightly smoke-test job I suspect it lists more then one. Did you try to parse the information into a json blob?

Reply:
Each smoke test is ran per artifact, the job to run the smoke test is within this block https://github.com/Azure/azure-sdk-for-python/blob/232b79b421199b53039134f03b61a2a5167e263e/eng/pipelines/templates/stages/archetype-python-release.yml#L14

Reply:
Here's where the smoke test is called https://github.com/Azure/azure-sdk-for-python/blob/232b79b421199b53039134f03b61a2a5167e263e/eng/pipelines/templates/stages/archetype-python-release.yml#L225

Reply:
What happens in the case of https://github.com/Azure/azure-sdk-for-python/blob/main/common/smoketest/smoke-test.yml?

Reply:
I triggered a test run https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1057590&view=results please have a look at those results to ensure it works the way we expect.

Reply:
It looks like this code might not be used for the daily smoke-test: `- ${{ if eq(parameters.Daily, false) }}:` so perhaps that isn't a valid case.

Reply:
When we start digging into the smoke-tests more we need to take a clearer look at how we expect to update the versions for the daily smoke-test.

Reply:
Yeah the nightly run runs another pip install command https://github.com/Azure/azure-sdk-for-python/blob/232b79b421199b53039134f03b61a2a5167e263e/eng/pipelines/templates/jobs/smoke.tests.yml#L195